GliaWidgets/View/Chat/ChatView.swift

```diff
@@ -101,8 +101,14 @@ class ChatView: EngagementView {
         updateTableView(animated: animated)
     }
 
-    func refreshSection(_ section: Int) {
-        tableView.reloadSections([section], with: .fade)
+    func refreshSection(_ section: Int, animated: Bool) {
```

Comment:
I didn't notice any usage of `animated: true` - did I miss something?

Reply:
Well, the initial implementation of this method behaved as if `animated` was set to `true`. That's why I left in the possibility to do so. But you're correct, currently, it is unused. Do you think I should just remove it altogether?

Reply:
I think you can currently remove it, yes.

Reply:
`animated` was removed. The function now always behaves as if it was set to `false`.
Couldn't think of a good use case for animated section reload, so I agree with you that it is not needed.